lib/europeana/page.rb

```diff
@@ -128,6 +140,10 @@ def breadcrumbs
       crumbs
     end
 
+    def page_and_ancestors
+      @page_and_ancestors ||= @page.self_and_ancestors
```

Comment:
Is it beneficial to assign `@page.self_and_ancestors` to a new instance variable, or could this method just return `@page.self_and_ancestors`?

Reply:
yes, because it's used to construct the breadcrumbs and to identify the exhibition. Previously these two lookups were made with different page depth specified as params of a where clause. This way we get the whole tree and can detect/select only the pages with the relevant depth.